app/Console/Commands/SendToQuasar.php

```diff
@@ -55,9 +57,14 @@ public function handle()
             $signups = $signups->where('updated_at', '<=', $end);
         }
 
-        $signups->chunkById(1000, function ($signups) {
+        $signups->chunkById(1000, function ($signups) use ($logFreq) {
             foreach ($signups as $signup) {
-                SendSignupToQuasar::dispatch($signup);
+                if ($signup->id % $logFreq === 0) {
+                    $log = true;
+                } else {
+                    $log = false;
```

Comment:
I think you could condense this into a one-liner... your call if that's readable or not! 😜

```php
$shouldWriteToLog = (bool) ($signup->id % $logFreq === 0);
SendSignupToQuasar::dispatch($signup, $shouldWriteToLog);
```

Reply:
i like it

Reply:
@DFurnes I like this especially because it works without the casting (like this:                 `$shouldWriteToLog = ($signup->id % $logFreq === 0);`) too! 🍲

Reply:
Oh right, even better!! 🎬